.travis.yml

```diff
@@ -103,8 +106,6 @@ notifications:
     on_success: always
 
 before_install:
-  # Ensure latest tags are available
-  - git fetch --tags
```

Comment:
I don't really understand versioneer, but I find it somewhat peculiar that that if you go to http://www.statsmodels.org/devel/, the version number it reports is `0+untagged.775.ga764908`. I would have thought its because we weren't getting tags and versioneer is tag-driven, but I don't know now. 

Any ideas how to get the correct version to show up?

Reply:
It needs the commit with the tag.  the depth was 50 and the tag was ~700 commits ago.  Setting depth to 10000 fixed it.

Reply:
doctr released tonight so one this PR is in the doc build should be all wrapped up.